apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingFactory.kt

```diff
@@ -217,3 +221,8 @@ class BehandlingFactory(
 }
 
 data class BehandlingOgOppgave(val behandling: Behandling, val oppgave: OppgaveIntern?)
+
+class ManuellMigreringHarEksisterendeIverksattBehandling : UgyldigForespoerselException(
+    code = "MANUELL_MIGRERING_EKSISTERENDE_IVERKSATT",
+    detail = "Eksisterer sak for søker der det allerede eksisterer en sak med en iverksatt behandling",
```

Comment:
Det eksisterer en sak for søker der det allerede eksistererer en sak for søker? :D